content/channels/index.textile

```diff
@@ -686,20 +674,19 @@ A client can subscribe to all messages published to a channel by passing a lambd
 The following is an example of registering a listener for all messages:
 
 ```[realtime_javascript]
-  var realtime = new Ably.Realtime('{{API_KEY}}');
-  var channel = realtime.channels.get('{{RANDOM_CHANNEL_NAME}}');
-  channel.subscribe(function(message) {
-    alert('Received: ' + message.data);
-  });
+const realtime = new Ably.Realtime.Promise('{{API_KEY}}');
+const channel = realtime.channels.get('{{RANDOM_CHANNEL_NAME}}');
+await channel.subscribe((message) => {
```

Comment:
It's maybe debatable, but I think we should probably not bother awaiting the call to `subscribe` here, it will more likely serve to confuse new users because it's not entirely obvious when the promise will resolve